diff -ur claws-mail-extra-plugins-3.7.7.orig//geolocation_plugin-0.0.4/configure.ac claws-mail-extra-plugins-3.7.7/geolocation_plugin-0.0.4/configure.ac
--- claws-mail-extra-plugins-3.7.7.orig//geolocation_plugin-0.0.4/configure.ac	2010-11-24 11:39:12.000000000 +0100
+++ claws-mail-extra-plugins-3.7.7/geolocation_plugin-0.0.4/configure.ac	2010-11-24 11:39:50.328687496 +0100
@@ -97,7 +97,7 @@
 PKG_CHECK_MODULES(LIBSOUP, libsoup-2.4)
 
 dnl Check for libchamplain and clutter
-PKG_CHECK_MODULES(CHAMPLAIN, champlain-gtk-0.4 >= 0.4.0 clutter-gtk-0.10)
+PKG_CHECK_MODULES(CHAMPLAIN, champlain-gtk-0.8 >= 0.8.0 clutter-gtk-0.10)
 
 AC_SUBST(VERSION)
 AC_SUBST(PLUGINVERSION)
